xds/internal/balancer/edsbalancer/balancergroup.go

```diff
@@ -31,6 +32,58 @@ import (
 	orcapb "google.golang.org/grpc/xds/internal/proto/udpa/data/orca/v1/orca_load_report"
 )
 
+type balancerConfig struct {
```

Comment:
Hmm ... I'm wondering if `balancerConfig` is the appropriate name for this. Would something like a `subBalancer` or a `childBalancer` be better? Because all this is, is a wrapper around the underlying balancer. Although `subBalancerWrapper` or `childBalancerWrapper` feels too long.

Reply:
Renamed to `subBalancerConfig`.